docs/content/users/configuration/config_yaml.md

```diff
@@ -1,75 +1,74 @@
+| upload_dir                       | Path from docroot to the upload directory targeted by `ddev import-files`                 |                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| working_dir                      | Working directories used by `ddev exec` and `ddev ssh`                                    | Example: `working_dir: { web: "/var/www", db: "/etc" }` sets the working directories for the `web` and `db` containers.                                                                                                                                                                                                                                                                                                                     |
+| omit_containers                  | Containers the project should not load                                                    | Can include `db`, `dba`, and `ddev-ssh-agent`.<br><br>Example: `omit_containers: [db, dba, ddev-ssh-agent]` starts the project without a `db` container, phpMyAdmin, and SSH agent. Some containers can be omitted globally in `~/.ddev/global_config.yaml` and the result is additive; all containers named in both places will be omitted.<br><br>Omitting the `db` container will cause database-dependent DDEV features to be unstable. |
+| web_environment                  | Additional custom environment variables for the project’s web container                   | Example: `web_environment: ["SOMEENV=someval", "SOMEOTHERENV=someotherval"]`.                                                                                                                                                                                                                                                                                                                                                               |
+| nfs_mount_enabled                | Whether to use NFS to mount the project into the container for performance                | Can be `true` or `false`, and requires host configuration before it can be used. See [nfs_mount_enabled documentation](../install/performance.md). Project-level configuration of this option is unusual, and when `true` in the global config it overrides the project-specific value.                                                                                                                                                     |
```

Comment:
```suggestion
| nfs_mount_enabled                | Whether to use NFS to mount the project into the container for performance                | Can be `true` or `false`, and requires host configuration before it can be used. See [nfs_mount_enabled documentation](../install/performance.md). Project-level configuration of this option is unusual, and when `true` in the global config it overrides the project-specific value. (Not commonly used now that much-faster Mutagen is available.)                                                                                                                                                    |
```